src/app/base/components/MainContentSection/MainContentSection.tsx

```diff
@@ -25,11 +25,11 @@ const MainContentSection = ({
     <div className="section" {...props} id="main-content-section">
       <div>
         {header ? (
-          <header className="section__header-wrapper">
```

Comment:
What's the reasoning for changing this from a semantic to non-semantic tag?

Reply:
https://warthogs.atlassian.net/browse/MAASENG-1263